src/components/sidenav/README.md

```diff
@@ -0,0 +1,66 @@
+# MdSidenav
+
+MdSidenav is the side navigation component for Material 2. It is composed of two components; `<md-sidenav-layout>` and `<md-sidenav>`.
+
+## `<md-sidenav-layout>`
+
+The parent component. Contains the code necessary to coordinate one or two sidenav and the backdrop.
+
+### Properties
+
+| Name | Description |
+| --- | --- |
+| `start` | The start aligned `MdSidenav` instance, or `null` if none is specified. In LTR direction, this is the sidenav shown on the left side. In RTL direction, it will show on the right. |
+| `end` | The end aligned `MdSidenav` instance, or `null` if none is specified. This is the sidenav opposing the `start` sidenav. |
+
+## `<md-sidenav>`
+
+The sidenav panel.
+
+### Inputs
```

Comment:
How would you feel about changing the `Input` / `Output` headers to `Bound properties` and `Events` ?